docs/sources/dashboards/share-dashboards-panels/index.md

```diff
@@ -54,6 +54,9 @@ Grafana enables you to share dashboards and panels with other users within an or
 - A Snapshot
 - An embedded link (for panels only)
 - An export link (for dashboards only)
+- A report
+
+<!-- has any of the information in these next 3 sentences changed? -->
```

Comment:
We have not touch permissions so these next three sentences should still stand as is